src/Util/Search.php

```diff
@@ -139,6 +139,30 @@ public function buildQuery(array $requestTerms): Query
         // Bool query to get range temporal extent dates
         $collectionDateBoolQuery = new Query\BoolQuery();
 
+        // Check if exclude term exists in the given query term
+        $mustNotQueryTerm = '';
+        if (preg_match_all('/-\b(\w*\w*)\b/', $queryTerm, $matches)) {
+            $mustNotQueryTerm = $matches[1][0];
```

Comment:
This only supports one word, I think it should support all words that start with `-`